### DISCUSSION TITLE
How to get around "No active editor, can't append templates."?

### DISCUSSION THREAD

**Discussion by itsz-san** [SilentVoid13/Templater#1353] [Help]
Maybe I'm missing something obvious, but how do we get around this? Some of my templates, have no need for an active editor.

For example, a "Create, New Note" template would simply create a new note from a template and has no need for an active editor.

But when trying to run the template from the "Templater: Open Insert Template modal" from a "No file is open" tab, it throws the error "No active editor, can't append templates."

**Comment by Zachatoo** [comment]
If you instead use the "Create new note from Template" command to create the note, then Templater does not require an active editor. You'll need to adjust your template to not create an additional file in this case. Otherwise I do not believe there is a workaround.
